src/typst.jl

```diff
@@ -159,7 +159,7 @@ function _showas(io::IO, ::MIME"text/typst", s::Subscript)
 end
 
 function _str_typst_escaped(io::IO, s::AbstractString)
-    escapable_special_chars = raw"\$#*_"
+    escapable_special_chars = raw"\$#*_<@\\`"
```

Comment:
is that at the end supposed to be an escaped backslash? As this is a raw string, the first backslash is actually a backslash and not an escape symbol for the $, so you don't need the addition

Reply:
Bitten by editor syntax highlighting there, will remove that `\\`.